### ISSUE TITLE
UI Update Identicons 

### ISSUE THREAD

**Issue by niklasp** [Proof-Of-Chaos/website#18]

**Comment by niklasp** [comment]
80$/hourly 3-4 hours integrating the subsocial sdk and writing components for wallet address display including placeholder and loading animations.

**Comment by niklasp** [comment]
<img width="888" alt="Bildschirmfoto 2022-10-08 um 15 28 34" src="https://user-images.githubusercontent.com/1685139/194709993-5274471a-1133-4486-bb6f-a126664555ed.png">

**Comment by gabrieljaegerde** [comment]
What about starting with on-chain identities? Anyone can easily set an identity for their wallet in polkadot.js. This might be more suitable than using subsocial ones (+it's also very easy to ingegrate)?

**Comment by niklasp** [comment]
yep thats probably what i mean :-) thought it was sub id but i probably set mine on polkadot as well

**Comment by gabrieljaegerde** [comment]
I think that would be cool. 

These three links will help you:
[An example of how to get all identities](https://stackoverflow.com/questions/66123619/how-to-find-all-the-polkadot-accounts-with-index-or-identity-attached
)
[An example of how to get one identity and decode it](https://github.com/polkadot-js/api/issues/2245)
[polkadotjs identity documentation](https://polkadot.js.org/docs/substrate/storage/#identity)
